reference-architecture/vmware-ansible/roles/static-ipaddress/tasks/main.yaml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
The initial deployment uses a DHCP address. The static address assigned allows for DNS resolution and the rest of the stuff required.